src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadSubmissionResource.java

```diff
@@ -147,7 +146,7 @@ public FileUploadSubmissionResource(SubmissionRepository submissionRepository, R
     }
 
     /**
-     * GET /file-upload-submissions/:id : get the fileUploadSubmissions by it's id. Is used by tutor when assessing submissions.
+     * GET /file-upload-submissions/:submissionId : get the fileUploadSubmissions by it's id. Is used by tutor when assessing submissions.
```

Comment:
"get the fileUploadSubmission by its id" (singular and its instead of it's). I think one could remove the second sentence, it is hard to maintain this information... it could be used on other occasions as well. The documentation should only include, what it does, not where it is used shouldn't it?

Reply:
edit: I just saw, that the description spans over the next two lines as well...maybe in this case, it is ok to add the description.... but it could also be put directly above the code that distinguish between the two use cases. 
in Line 150, it should be "in case an instructor calls..." (singular)